File: C++/Programming Basics with C++/07. Exercise - Nested Conditional Statement/simple_task/Journey.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    double budget;
    string season;
    cin >> budget >> season;

    string destination;
    string place;
    double totalPrice;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (season == "summer") {
            place = "Camp";
            totalPrice = budget * 0.3;
        } else {
            place = "Hotel";
            totalPrice = budget * 0.7;
        }
    } else if (budget <= 1000) {
        destination = "Balkans";
        if (season == "summer") {
            place = "Camp";
            totalPrice = budget * 0.4;
        } else {
            place = "Hotel";
            totalPrice = budget * 0.8;
        }
    } else {
        destination = "Europe";
        place = "Hotel";
        totalPrice = budget * 0.9;
    }

    cout.setf(ios::fixed);
    cout.precision(2);

    cout << "Somewhere in " << destination << endl;
    cout << place << " - " << totalPrice << endl;

    return 0;
}

File: C#/Programming Basics with C#/07. Nested Conditional Statements - Exercise/08. Journey/Journey.cs
using System;

class Journey
{
    static void Main()
    {
        double buget = double.Parse(Console.ReadLine());
        string sesson = Console.ReadLine().ToLower();

        if (buget <= 100)
        {
            Console.WriteLine("Somewhere in Bulgaria");
            if (sesson == "summer")
            {
                buget = buget * 0.30;
                Console.WriteLine($"Camp - {buget:F2}");
            }
            else if (sesson == "winter")
            {
                buget = buget * 0.70;
                Console.WriteLine($"Hotel - {buget:F2}");
            }
        }
        else if (buget <= 1000)
        {
            Console.WriteLine("Somewhere in Balkans");
            if (sesson == "summer")
            {
                buget = buget * 0.40;
                Console.WriteLine($"Camp - {buget:F2}");
            }
            else if (sesson == "winter")
            {
                buget = buget * 0.80;
                Console.WriteLine($"Hotel - {buget:F2}");
            }
        }
        else if (buget > 1000)
        {
            Console.WriteLine("Somewhere in Europe");
            buget = buget * 0.90;
            Console.WriteLine($"Hotel - {buget:F2}");
        }
    }
}